observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,79 @@ The Network Observability Operator enables administrators to observe and analyze
 These release notes track the development of the Network Observability Operator in the {product-title}.
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+
+[id="network-observability-operator-release-notes-1-8_{context}"]
+== Network Observability Operator 1.8.0
+The following advisory is available for the Network Observability Operator 1.8.0:
+
+* link:https://access.redhat.com/errata/RHSA-2024:8014[Network Observability Operator 1.8.0]
+
+[id="network-observability-operator-1.8.0-features-enhancements_{context}"]
+=== New features and enhancements
+
+[id="network-observability-operator-pkt-xlat-1-8_{context}"]
+==== Packet translation
+You can now enrich network flows with translated endpoint information, showing not only the service but also the specific backend pod, so you can see which pod served a request.
+
+[id="network-observability-operator-OVN-observability-1-8_{context}"]
+==== OVN-Kubernetes networking events tracking
+
+:FeatureName: OVN-Kubernetes networking events tracking
+include::snippets/technology-preview.adoc[]
+
+You can now use network event tracking in Network Observability to gain insight into OVN-Kubernetes events, including network policies, admin network policies, and egress firewalls.
+
```

Comment:
as discussed, we should also mention [performance improvements](https://netobserv.io/posts/performance-improvements-in-1-8/) of the eBPF agent.
If you want a super-quick summary you can tell that we've observed CPU reduction between -40% and -57% in the eBPF Agent.

Reply:
Added a section for this, so marking as resolved.